input/region_tail.xml.in

```diff
@@ -0,0 +1,7 @@
+
+      	</initialfacilitylist>
+      <config><NullInst/></config>
+    </institution>
+  </region>
+
```

Comment:
Yeah I definitely saw that in the PEP8 it was like a pet peeve, but I found it easier to
read the compiled input file if I add a space in the tail.
Do you think that is worth it though?